lighthouse-core/computed/resource-summary.js

```diff
@@ -102,16 +102,16 @@ class ResourceSummary {
   }
 
   /**
-   * @param {{URL: LH.Artifacts['URL'], devtoolsLog: LH.DevtoolsLog}} data
-   * @param {LH.Audit.Context} context
+   * @param {{URL: LH.Artifacts['URL'], devtoolsLog: LH.DevtoolsLog, settings: ImmutableObject<LH.Config.Settings>}} data
+   * @param {LH.Gatherer.ComputedContext} context
    * @return {Promise<Record<LH.Budget.ResourceType,ResourceEntry>>}
    */
   static async compute_(data, context) {
```

Comment:
do you think this should just accept a budget at this point rather than all of settings?

Reply:
I'm going to make this change.

Only problem I found was in the test file for `computed/resource-summary.js`. The tests expect the first arg to be a subset of `artifacts`. Making this change disrupts that naming convention.